app/src/main/java/com/mxt/anitrend/base/custom/activity/ActivityBase.java

```diff
@@ -183,7 +183,7 @@ public boolean onCreateOptionsMenu(Menu menu) {
     @Override
     public boolean onOptionsItemSelected(MenuItem item) {
         if (item.getItemId() == android.R.id.home)
-            super.onBackPressed();
+            onBackPressed();
```

Comment:
had to remove super here otherwise, NotificationActivity::onBackPressed() was not being called.

Reply:
Well, with super here it'll call the original onBackPressed() rather than a local one. It'll never reach NotificationActivity in this case.
I can relocate super.onBackPressed() call tho.

Reply:
You're absolutely right, I was think of this in the wrong order.